Source/Core/AudioCommon/AudioCommon.cpp

```diff
@@ -192,7 +193,7 @@ void SetSoundStreamRunning(Core::System& system, bool running)
 
 void SendAIBuffer(Core::System& system, const short* samples, unsigned int num_samples)
 {
-  SoundStream* sound_stream = system.GetSoundStream();
+  const SoundStream* sound_stream = system.GetSoundStream();
```

Comment:
You might as well make the variable itself `const` if you are making things `const`.
```suggestion
  const SoundStream* const sound_stream = system.GetSoundStream();
```
There's similar situations in a few other places that you touched.